.github/stale.yml

```diff
@@ -24,18 +24,27 @@ staleLabel: Review - Abandoned
 # Comment to post when marking as stale. Set to `false` to disable
 markComment: >
   This PR has been automatically marked as stale because it has not had activity in the last 30 days.
+
```

Comment:
What I meant was that these should not be on separate lines in the message. Only the final note about being up for grabs should be on a new line.